### PR TITLE
Update NexentaStor version and supported features

### PR DESCRIPTION
**Pull request by Qeas**
Added v1.2, v1.3, v1.4 and Topology support

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [MooseFS](https://github.com/moosefs/moosefs-csi) | `com.tuxera.csi.moosefs` | v1.0 | A Container Storage Interface (CSI) Driver for [MooseFS](https://moosefs.com/) clusters. | Persistent | Read/Write Multiple Pods | Yes | 
 [NetApp](https://github.com/NetApp/trident) | `csi.trident.netapp.io` | v1.0, v1.1 | A Container Storage Interface (CSI) Driver for NetApp's [Trident](https://netapp-trident.readthedocs.io/) container storage orchestrator | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning
-[NexentaStor](https://github.com/Nexenta/nexentastor-csi-driver) | `nexentastor-csi-driver.nexenta.com` | v1.0, v1.1 | A Container Storage Interface (CSI) Driver for NexentaStor | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning
+[NexentaStor](https://github.com/Nexenta/nexentastor-csi-driver) | `nexentastor-csi-driver.nexenta.com` | v1.0, v1.1, v1.2, v1.3, v1.4 | A Container Storage Interface (CSI) Driver for NexentaStor | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning, Topology
```
Comment (xing-yang) [inline book/src/drivers.md:62]: The 3rd column is for compatible CSI spec versions, not CSI driver version.  The highest CSI spec version is v1.2 at the moment.

Comment (Qeas) [reply book/src/drivers.md]: fixed

**Comment by xing-yang** [comment]
Looks good now.  Can you squash your commits into 1?

**Comment by Qeas** [comment]
> Looks good now. Can you squash your commits into 1?

Done.

**Pull request merged**